src/lib.rs

```diff
@@ -477,6 +497,40 @@ impl Criterion {
         self
     }
 
+    /// Benchmarks multiple functions
+    ///
+    /// All functions get the same input and are compared with the other implementations.
+    /// Works similar to `bench`, but with multiple functions.
+    ///
+    /// ``` ignore-test
+    /// fn bench_seq_fib(b: &mut Bencher, i: &u32) {
+    ///     b.iter(|| {
+    ///         seq_fib(test::black_box(i));
```

Comment:
We make it cleaner without `black_box` so I vote for the path where we document without it and say in some caveat section that if you get ridiculous performance you should probably add some `black_box`es :)
I don't have any concrete proof or example where a `black_box` is needed on the user level, so I might be wrong, thus it's better to document without it as well. I'll fix this when I have the time.